src/Symfony/Component/Lock/BlockingStoreInterface.php

```diff
@@ -22,6 +23,7 @@ interface BlockingStoreInterface extends PersistingStoreInterface
      * Waits until a key becomes free, then stores the resource.
      *
      * @throws LockConflictedException
+     * @throws LockStorageException
```

Comment:
Which implementation throw that exception?

Reply:
Hi,

Just here : https://github.com/symfony/lock/blob/7.0/Store/FlockStore.php#L109

Reply:
But is this a case where we would expect the caller to catch this exception ? If not (i.e. you would let it reach the global exception handler anyway), this `@throws` brings no value at all.

Reply:
I agree with @stof here.